UPDATING.md

```diff
@@ -25,6 +25,8 @@ assists people when migrating to a new version.
 
 * [11244](https://github.com/apache/incubator-superset/pull/11244): The `REDUCE_DASHBOARD_BOOTSTRAP_PAYLOAD` feature flag has been removed after being set to True for multiple months.
 
+* [11098](https://github.com/apache/incubator-superset/pull/11098): includes a database migration that adds a column caled `uuid` to all models, and updates the `Dashboard.position_json` column to also have UUIDs of charts. Depending on the number of objects the migration might take up to 3-4 minutes, requiring planning for the downtime.
```

Comment:
NIT: "to all models" -> "to some models" (?)

Reply:
nit: I'm no native speaker, but the two "the"'s in the last sentence seems not necessary?